src/elementary/IsotopeID/test/IsotopeID.test.cpp

```diff
@@ -19,21 +19,37 @@ SCENARIO( "IsotopeID" ) {
       IsotopeID id( 1, 1 );
       CHECK( ElementID( 1 ) == id.element() );
       CHECK( 1 == id.mass() );
+      CHECK( "H1" == id.symbol() );
       CHECK( "H1" == id.name() );
 
       id = IsotopeID( 1001 );
       CHECK( ElementID( 1 ) == id.element() );
       CHECK( 1 == id.mass() );
+      CHECK( "H1" == id.symbol() );
+      CHECK( "H1" == id.name() );
+
+      id = IsotopeID( "H1" );
+      CHECK( ElementID( 1 ) == id.element() );
+      CHECK( 1 == id.mass() );
+      CHECK( "H1" == id.symbol() );
       CHECK( "H1" == id.name() );
 
       id = IsotopeID( 1, 0 );
       CHECK( ElementID( 1 ) == id.element() );
       CHECK( 0 == id.mass() );
+      CHECK( "H0" == id.symbol() );
       CHECK( "H0" == id.name() );
 
       id = IsotopeID( 1000 );
       CHECK( ElementID( 1 ) == id.element() );
       CHECK( 0 == id.mass() );
+      CHECK( "H0" == id.symbol() );
+      CHECK( "H0" == id.name() );
+
+      id = IsotopeID( "H0" );

```

Comment:
In its current form it is not allowed, but I would actually to allow it. I understand why the zero is required though: it would be confusing with the actual element identifier.